### ISSUE TITLE
Restore just-include argument

### ISSUE THREAD

**Issue by timangus** [joncloud/nsis-action#5]
I use NSIS in a project that I'm currently migrating to GH Actions from Travis. It uses NSIS to build installers, and is invoked from a batch script. In other words, I don't want or need an action that runs makensis, only one that installs it. This is possible with v1 of your action, with just-include set to true. This option is not available in v2 however. For my use case it's basically crucial. Please consider reinstating it, thanks.

**Comment by joncloud** [comment]
Thanks for the feedback. I'll look into the best way to incorporate the capability.

**Comment by joncloud** [comment]
I have made a tentative release, `v3.0`, which removes nsis from the repository altogether. GitHub Actions [Windows virtual environments](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#tools) currently have NSIS v3.06.1 already installed on their images. It can be found at `C:\Program Files (x86)\NSIS\makensis.exe`.

My recommendation if you are using a self-hosted runner would be one of the following:
* Leverage `joncloud/makensis-action@v1.2` to keep the original API for producing the `makensis.exe` path.
* Maintain and install a copy of NSIS on your self-hosted runner.
* Automate installation of NSIS with additional GitHub Action Workflows.

I'll revisit, and close this once I have time to perform additional tests in the [makensis-action-test](https://github.com/joncloud/makensis-action-test) repo.

**Comment by timangus** [comment]
Huh, I could swear I checked if the Windows virtual environment already had NSIS installed and came up blank. Looking at the history of the README it appears like it was added a couple of months ago, but I have checked since then. Oh well, turns out I don't need your action at all then. Apologies if I wasted your time at all, though I guess it's good change to avoid including NSIS if you don't need to.
